compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/Types.java

```diff
@@ -6276,6 +6276,7 @@ private boolean isServiceObject(BType bType) {
     public static class IntersectionContext {
         Location lhsPos;
         Location rhsPos;
+        Location pos;
```

Comment:
Let's add a comment indicating what this represents.

Reply:
Added